### PR TITLE
Release flink 1.15.2

### PR DESCRIPTION
**Pull request by dannycranmer**
Preparing for the 1.15.2 release.

### CODE REVIEW

File: _posts/2022-08-17-release-1.15.2.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+categories: news
+authors:
+- danny:
+  name: "Danny Cranmer"
+
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink 1.15.
+
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink 1.15 series.
```
Comment (zentol) [inline _posts/2022-08-17-release-1.15.2.md]:
```suggestion
The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.15 series.
```

File: _posts/2022-08-17-release-1.15.2.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27199'>FLINK-27199</a>] -         Bump Pulsar to 2.10.0 for fixing the unstable Pulsar test environment.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27865'>FLINK-27865</a>] -         Add guide and example for configuring SASL and SSL in Kafka SQL connector document
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28094'>FLINK-28094</a>] -         Upgrade AWS SDK to support ap-southeast-3 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28140'>FLINK-28140</a>] -         Improve the documentation by adding Python examples
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28329'>FLINK-28329</a>] -         List top 15 biggest directories in terms of used disk space
+</li>
```
Comment (zentol) [inline _posts/2022-08-17-release-1.15.2.md]:
```suggestion
```
internal stuff

File: _posts/2022-08-17-release-1.15.2.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-26600'>FLINK-26600</a>] -         PerJobMiniClusterFactoryTest.testJobClientSavepoint failed on azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-26621'>FLINK-26621</a>] -         ChangelogPeriodicMaterializationITCase crashes JVM on CI in RocksDB cleanup
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28263'>FLINK-28263</a>] -         TPC-DS Bash e2e tests don&#39;t clean-up after completing
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28680'>FLINK-28680</a>] -         No space left on device on Azure e2e_2_ci tests
+</li>
+</ul>
```
Comment (zentol) [inline _posts/2022-08-17-release-1.15.2.md]: > (excluding improvements to the build infrastructure and build stability)
```suggestion
```

**Review by zentol — APPROVED** [review]

File: _posts/2022-08-17-release-1.15.2.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink?tab=tags&page=1&name=1.15.2) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=1.15.2) (ASF repository)
+
+## PyPi
+
+* [apache-flink==1.15.1](https://pypi.org/project/apache-flink/1.15.2/)
```
Comment (HuangXingBo) [inline _posts/2022-08-17-release-1.15.2.md]:
```suggestion
* [apache-flink==1.15.2](https://pypi.org/project/apache-flink/1.15.2/)
```

File: _posts/2022-08-17-release-1.15.2.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-25815'>FLINK-25815</a>] -         PulsarSourceITCase.testTaskManagerFailure failed on azure due to timeout
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-25884'>FLINK-25884</a>] -         PulsarSourceITCase.testTaskManagerFailure failed on azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-26177'>FLINK-26177</a>] -         PulsarSourceITCase.testScaleDown fails with timeout
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-26721'>FLINK-26721</a>] -         PulsarSourceITCase.testSavepoint failed on azure pipeline
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27833'>FLINK-27833</a>] -         PulsarSourceITCase.testTaskManagerFailure failed with AssertionError
```
Comment (HuangXingBo) [inline _posts/2022-08-17-release-1.15.2.md]: These issues are unstable tests about Pulsar, maybe not listed here?

**Review by HuangXingBo — APPROVED** [review]

File: _posts/2022-08-17-release-1.15.2.md (hunk 5 above)
Comment (dannycranmer) [reply _posts/2022-08-17-release-1.15.2.md]: Thanks, I have removed the IT test failure fix issues from the release notes

**Review by dawidwys — APPROVED** [review]

**Pull request merged**